Source/town.cpp

```diff
@@ -130,9 +114,6 @@ void TownCloseHive()
 	SetDungeonMicros();
 }
 
-/**
- * @brief Update the map to show the closed grave
- */
 void TownCloseGrave()

```

Comment:
TownCloseGrave() is a function local to the cpp unit, it is not declared elsewhere and so it's documentation should not be moved.

Reply:
Okay, now it's more clear for me. Thanks.